core/main.ts

```diff
@@ -135,12 +136,28 @@ function loadActionConfigs(session: Session, filePaths: string[]) {
           actionConfig.target.name = fileNameAsTargetName;
         }
 
+        // TODO(ekrekr): throw an error if incorrect configs are specified for the filetype.
+        // TODO(ekrekr): add a test for nice errors if files are not found.
+
         if (fileExtension === "ipynb") {
-          // TODO(ekrekr): throw an error if any non-notebook configs are given.
-          // TODO(ekrekr): add test for nice errors if file not found.
           const notebookContents = nativeRequire(actionConfig.fileName).asBase64String();
           session.notebook(dataform.ActionConfig.create(actionConfig), notebookContents);
         }
+
+        if (fileExtension === "sql") {
```

Comment:
just checking, in the case of action config we only expect sql nor sqlx?

Reply:
Yep! SQLX is still handled here https://github.com/dataform-co/dataform/blob/cab21959a1857f964b113e94d9a2d3b8fbc51c0a/core/main.ts#L83